CWE/decisions/Algeria.txt

```diff
@@ -55,11 +55,10 @@ political_decisions = {
 			tag = FAG
 		}
 		allow = {
-			OR = { government = democracy government = democracy1 }
```

Comment:
Only a democracy should be able to switch French Algeria into normal Algeria. This is as the latter is meant to represent a multicultural Algeria being governed by non whites which is something the whites would not allow such a transition if they were running a dictatorship.